run.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# run.sh - Runs everything necessary to generate our API artifacts (for
+# website, external consumers, etc.) based on our inputs (from
+# covid-data-public).
+
+set -o nounset
+set -o errexit
+
+DEFAULT_DATA_SOURCES_DIR="../covid-data-public"
+DEFAULT_API_RESULTS_DIR="api_results"
+
+main () {
+  # Go to repo root (where run.sh lives).
+  cd "$(dirname "$0")"
```

Comment:
you could do this using `pushd` and `popd` so that it doesn't change your directory after the run

Reply:
`cd` within a bash script doesn't affect the current directory of the caller.  So `pushd` / `popd` isn't necessary.

Reply:
time to go brush up on my bash 😀